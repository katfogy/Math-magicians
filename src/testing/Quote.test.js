import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
// eslint-disable-next-line import/no-extraneous-dependencies
import renderer from 'react-test-renderer';
import Quotes from '../components/quotes'

describe('Quotes', () => {
  it('Should render correctly', () => {
    const { getByText } = render(<Quotes />);
    expect(getByText('Loading...')).toBeInTheDocument();
  });
  it('renders correctly', () => {
    const tree = renderer.create(<Quotes />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});