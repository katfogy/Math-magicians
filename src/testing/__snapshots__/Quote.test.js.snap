// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quotes renders correctly 1`] = `
<div
  className="container"
>
  <ul />
</div>
`;
